src/bridgestan.h

```diff
@@ -18,9 +18,9 @@ extern int bs_patch_version;
  * generator (PRNG) wrapper.  Data must be encoded in JSON as
  * indicated in the *CmdStan Reference Manual*.
  *
- * @param[in] data_file C-style string. This is either a
- * path to JSON-encoded data file (must end with ".json"), or
- * a JSON string literal.
+ * @param[in] data C-style string. This is either a
+ * path to JSON-encoded data file (must end with ".json"),
+ * a JSON string literal, or nullptr, which is interpreted as no data.
```

Comment:
```suggestion
 * a JSON string literal, an empty string, or nullptr, which is interpreted as no data.
```

Reply:
Agreed, it is good to call out this case specifically.